### PR TITLE
Create Aine filia Saoirse

### PR DESCRIPTION
**Pull request by nightbe**
I dont know if i have made this correctly. I hope so. Its Kristine from the discord server.

### CODE REVIEW

**Comment by GuillaumeDIDIER** [comment]
First glance looks good. I’ll double check things tomorrow and merge.

Thanks and congrats !!!

**Comment by GuillaumeDIDIER** [comment]
It looks to me like the file is missing the .md extension, please fix that :) Otherwise it looks good to me

File: Magi/Aine filia Saoirse [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
++ Infernal Lore +1 (Demons)
++ Language (native): Celtic 5 (Storytelling)
++ Language: Latin 4 (Academic use)
++ Language: Middle Dutch 4 (Arnhem Dialect)
++ Magic Lore 1 
++ Magic Theory +5 (Creo)
++ Parma Magica +2 (Ignem)
++ Penetration +3 (Ignem)
++ Single Weapons +2 (Short Sword)
++ Stealth +1 (Shadow)
```
Comment (GuillaumeDIDIER) [inline Magi/Aine filia Saoirse]: No need for '+' in front of the level. (Parma +2 (Ignem) -> Parma (Ig) 2)

Also I've tended to add the specialty before the level

**Comment by nightbe** [comment]
Hey :) I think i fixed it :)

**Comment by GuillaumeDIDIER** [comment]
Yep, this lady commit fixed it !

**Pull request merged**